ui/tract-renderer/src/test/java/org/cru/godtools/tract/util/DeepLinkUtilsTest.kt

```diff
@@ -0,0 +1,34 @@
+package org.cru.godtools.tract.util
+
+import android.app.Activity
+import android.content.Context
+import android.net.Uri
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import org.junit.Assert.assertFalse
+import org.junit.Assert.assertTrue
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.Robolectric
+
+@RunWith(AndroidJUnit4::class)
+class DeepLinkUtilsTest {
+    lateinit var context: Context
+
+    @Before
+    fun setup() {
+        context = Robolectric.buildActivity(Activity::class.java).get()
+    }
+
+    @Test
+    fun verifyIsTractDeepLink() {
+        assertFalse(Uri.parse("wss://knowgod.com/en/kgp").isTractDeepLink(context))
+        assertFalse(Uri.parse("https://example.com/en/kgp").isTractDeepLink(context))
+        assertFalse(Uri.parse("https://knowgod.com/en").isTractDeepLink(context))
+        assertTrue(Uri.parse("http://knowgod.com/en/kgp").isTractDeepLink(context))
```

Comment:
We use ```assertTrue``` when we expect the value to be true and vice-versa, right?

Reply:
yeah, there is a whole set of `assert*()` methods that allow you to test various things. It's part of the JUnit 4 framework

Reply:
we talked about testing frameworks, once you work on a task that requires some unit tests you'll get experience with this